package.json

```diff
@@ -21,10 +25,17 @@
     "react-helmet": "5.2.1"
   },
   "devDependencies": {
+    "@babel/core": "^7.9.0",
```

Comment:
I recommend to use exact version numbers:
`7.9.0` instead of `^7.9.0`
Just to reduce the risk to get broken builds.

Reply:
can be done later...